Framework/Kernel/src/Unit.cpp

```diff
@@ -583,6 +583,54 @@ double dSpacing::conversionTOFMax() const { return DBL_MAX / factorTo; }
 
 Unit *dSpacing::clone() const { return new dSpacing(*this); }
 
+// ==================================================================================================
+/* D-SPACING Perpendicular
+ * ==================================================================================================
+ *
+ * Conversion uses equation: d^2 = lambda^2 - 2[Angstrom^2]*ln(sin(theta))
```

Comment:
For me it is also unusual. The idea was to create a unit which is orthogonal to dSpacing(theta, lambda). This is needed for 2D Rietveld refinement of TOF powder diffraction data from POWTEX@MLZ and DREAM@ESS due to specific detector construction. We plan to test the approach at POWGEN soon.